test/sequential/test-chdir.js

```diff
@@ -38,3 +38,7 @@ assert(process.cwd() == dir);
 process.chdir('..');
 assert(process.cwd() == path.resolve(common.fixturesDir));
 fs.rmdirSync(dir);
+
+assert.throws('process.chdir({})', TypeError, 'Bad argument.');
+assert.throws('process.chdir()', TypeError, 'Bad argument.');
+assert.throws('process.chdir("x", "y")', TypeError, 'Bad argument.');
```

Comment:
This is both a V8-ism that works by accident and a bug in the assert module... assert.throws() calls the first argument as a function.  The result is a TypeError when the input is a string but it doesn't seem to check that the exception message matches...